### PR TITLE
(Solved issue #755 for develop) Update Copyright Year

### LINKED ISSUE #755: update copyright year 2020 to 2021
update of all our copyright strings

if noone grabs this I do this before the next release

### PR DESCRIPTION
**Pull request by Jatin2020-24**
Fixes : #755

### CODE REVIEW

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
please make a PR to stable.

**Review by joernu76 — DISMISSED** [review]

**Comment by joernu76** [comment]
> please make a PR to stable.

Isn't this fine for develop?

issue was for stable. but hey yeah. one day develop will be stable again.

File: mslib/msui/_tests/test_editor.py [hunk 1]
```diff
@@ -9,7 +9,7 @@
     This file is part of mss.
 
     :copyright: Copyright 2020 Vaibhav Mehra <veb7vmehra@gmail.com>
-    :copyright: Copyright 2020 by the mss team, see AUTHORS.
+    :copyright: Copyright 2021 by the mss team, see AUTHORS.
```
Comment (ReimarBauer) [inline mslib/msui/_tests/test_editor.py]: the change should here be 2020-2021

File: mslib/msui/_tests/test_mscolab_version_history.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
     This file is part of mss.
 
     :copyright: Copyright 2020 Tanish Grover
-    :copyright: Copyright 2019-2020 by the mss team, see AUTHORS.
+    :copyright: Copyright 2019-2021 by the mss team, see AUTHORS.
```
Comment (ReimarBauer) [inline mslib/msui/_tests/test_mscolab_version_history.py]: this got last time already wrong, 2020-2021

File: mslib/msui/editor.py [hunk 3]
```diff
@@ -9,7 +9,7 @@
     This file is part of mss.
 
     :copyright: Copyright 2020 Vaibhav Mehra <veb7vmehra@gmail.com>
-    :copyright: Copyright 2020 by the mss team, see AUTHORS.
+    :copyright: Copyright 2021 by the mss team, see AUTHORS.
```
Comment (ReimarBauer) [inline mslib/msui/editor.py]: 2020 - 2021

**Review by ReimarBauer — CHANGES_REQUESTED** [review]
We can merge that also to develop, there are a few Copyright 2020 which were not replaced to 2020 - 2021

see comments

File: mslib/msui/_tests/test_editor.py (hunk 1 above)
Comment (Jatin2020-24) [reply mslib/msui/_tests/test_editor.py]: I have changed the year.

File: mslib/msui/_tests/test_mscolab_version_history.py (hunk 2 above)
Comment (Jatin2020-24) [reply mslib/msui/_tests/test_mscolab_version_history.py]: Year changed

File: mslib/msui/editor.py (hunk 3 above)
Comment (Jatin2020-24) [reply mslib/msui/editor.py]: Year changed

**Comment by Jatin2020-24** [comment]
@ReimarBauer I have updated copyright year as per your guidance.

**Review by ReimarBauer — APPROVED** [review]
thx

**Pull request merged**